app/pages/home-for-user/circle-ribbon.jsx

```diff
@@ -168,6 +170,8 @@ const CircleRibbon = React.createClass({
       tooltipPosition = this.getTooltipPoint(hoveredArc, 50);
     }
 
+    const avatar = this.props.user.avatar_src ? this.props.user.avatar_src : '/assets/simple-avatar.png';
```

Comment:
The default will only be used while the user prop is null, basically, while we're waiting on authentication. This will set it to the fallback simple avatar if specifically the `avatar_src` for the current logged in user is null.